### ISSUE TITLE
Basic explanation of SQL required in Introduction to SQL lesson [Suggestion]

### ISSUE THREAD

**Issue by nshozi** [datacarpentry/sql-ecology-lesson#180]
[Instructor Training Suggestion]

In the SQL introduction episode ( sql-ecology-lesson/_episodes/00-sql-introduction.md ) , one of the questions/objectives for the course is to answer 'what is SQL' however when browsing this material, there is no section that defines SQL. I would like to suggest the following:

**What is SQL?**
SQL stands for Structured Query Language. It is a language that allows you to manipulate databases through queries. These queries can allow you to perform a number of action such as: insert a record into a database, update a record in a database, delete a record in a database and many others.

**Comment by orchid00** [comment]
Nshozi, thanks for your comment it's great to add! Can we maybe rephrase to SQL allows us to interact with relational databases. Instead of manipulate and then instead of talking about records, just link this to the existing concept of relational databases. We can polish this definition more comments are welcome.

**Comment by nshozi** [comment]
@orchid00 That really sounds much easier and simpler to understand - perfect!!

**Comment by orchid00** [comment]
@nshozi  just reviewing this again. Would you be able to put together the rephrased concept as a Pull request? So we can close this issue and have an updated concept! Thanks

**Comment by orchid00** [comment]
@nshozi I have added a concept based on the discussion to https://github.com/datacarpentry/sql-ecology-lesson/blob/gh-pages/_episodes/00-sql-introduction.md
Thanks for this thread.
